handlebars/.gitignore

```diff
@@ -1,28 +1,16 @@
-# Numerous always-ignore extensions
```

Comment:
The recommendation (of GitHub and others) is to add editor-specific ignores to a global ignore file. see https://github.com/github/gitignore/tree/master/Global#globally-useful-gitignores for details.

The per-project `.gitignore` file should only contain entries related to the project itself and its build system etc.